File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.kt
class Solution {
    fun solution(s: String, n: Int): String {
        var arr = s.toCharArray()
        var answer = ""
        for(i in 0 until arr.size){
            if(arr[i]==' '){ 
                answer += ' '
                continue
            }
            if('a'<=arr[i]&&arr[i]<='z'){
                if((arr[i]+n)>'z') answer += 'a'+(n-1-'z'.toInt()+arr[i].toInt())
                else answer += arr[i]+n
            }
            else{
                if((arr[i]+n)>'Z') answer += 'A'+(n-1-'Z'.toInt()+arr[i].toInt())
                else answer += arr[i]+n
            }
        }

        return answer
    }
}

File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = []
    s=list(s)
    for i in s:
        if i == " ":
            answer.append(" ")
            continue
        if ord(i)>=66 and ord(i)<=90: 
            if ord(i)+n>90:
                answer.append(chr(ord(i)+n-26))
                continue
            else:
                answer.append(chr(ord(i)+n))
        else:
            if ord(i)+n>122:
                answer.append(chr(ord(i)+n-26))
                continue
            else:
                answer.append(chr(ord(i)+n))
    return ''.join(answer)